### PR TITLE
[ISSUE-611] Update issue templates

### LINKED ISSUE #611: Update issue templates
Existing issue template don't have bug/feature separate and overloaded by details.

### PR DESCRIPTION
**Pull request by mishoyama**
### Resolves #611  

Update issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```
Comment (safronovD) [inline .github/ISSUE_TEMPLATE/feature_request.md:16]: Shouldn't it be optional?

**Review by safronovD — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (mishoyama) [reply .github/ISSUE_TEMPLATE/feature_request.md:16]: This is standard template. There are no hard requirements here

Comment (safronovD) [reply .github/ISSUE_TEMPLATE/feature_request.md:16]: Got it

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Environment (please complete the following information):**
+ - Hardware: [e.g. Dell PowerEdge]
+ - OS: [e.g. Ubuntu]
+ - Version: [e.g. 1.0.0]
```
Comment (rkiselenko) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: Can we use actual OS output `printf "$(uname -srm)\n$(cat /etc/os-release)\n"` ? 

Output example:
```
Linux 5.10.16.3-microsoft-standard-WSL2 x86_64
NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.3 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

**Review by rkiselenko — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mishoyama) [reply .github/ISSUE_TEMPLATE/bug_report.md:16]: Sure

**Pull request merged**